### ISSUE TITLE
Issue with highlightlines in minted v3

### ISSUE THREAD

**Issue by jakobmoss** [gpoore/minted#410]
First of all, excellent work on the new release and the package in general!

However, switching to version 3 seems to break some of my existing code, which relies on highlighting multiple bands of lines  in the same code block (following the package documentation). As an example, the following code (sorry for the Danish!) works with the old version of the package:
```latex
\begin{minted}[bgcolor=codebg,highlightlines={3-6,9-11}]{latex}
\begin{enumerate}
  \item Morgenmad
    \begin{itemize}
      \item Æg
      \item Bacon
    \end{itemize}
  \item Frokost
  \item Aftensmad
    \begin{enumerate}
      \item Grøntsager
    \end{enumerate}
\end{enumerate}
\end{minted}
```

However, trying with version 3, I am met with the following error:
```
! Package keyval Error: 9-11 undefined.

See the keyval package documentation for explanation.
Type  H <return>  for immediate help.
 ...

l.968 ...debg,highlightlines={3-6,9-11}]{latex}
``` 

It also happens for external files with e.g. `\inputminted[ ..., highlightlines={1,12}]{latex}{...}`, where I get `! Package keyval Error: 12 undefined.`.

Reverting to `\usepackage{minted2}` makes things work again. Which is excellent and I will just keep using that for now.

Am I missing something in how to specify highlighting of two line groups, or is it a bug?

Thanks again!

**Comment by muzimuzhi** [comment]
Should be fixed by https://github.com/gpoore/minted/pull/407.

Applying the patch in #407, your example compiles without errors.
```tex
\documentclass{article}
\usepackage{minted}

\usepackage{xcolor}
\colorlet{codebg}{yellow!20}

\begin{document}
\begin{minted}[bgcolor=codebg,highlightlines={3-6,9-11}]{latex}
\begin{enumerate}
  \item Morgenmad
    \begin{itemize}
      \item Æg
      \item Bacon
    \end{itemize}
  \item Frokost
  \item Aftensmad
    \begin{enumerate}
      \item Grøntsager
    \end{enumerate}
\end{enumerate}
\end{minted}
\end{document}
```
![image](https://github.com/user-attachments/assets/e2d77547-619a-4145-a582-4c7fd68b6c6d)

**Comment by gpoore** [comment]
This works correctly in the current release and was fixed by #407.
